code/modules/surgery/bodyparts/parts.dm

```diff
@@ -69,6 +69,17 @@
 		cavity_item = null
 	return ..()
 
+/// Sprite to show for photocopying mob butts
+/obj/item/bodypart/chest/proc/get_butt_sprite()
+	if(!ishuman(owner))
+		return null
+	var/mob/living/carbon/human/human_owner = owner
+	var/butt_sprite = human_owner.physique == FEMALE ? BUTT_SPRITE_HUMAN_FEMALE : BUTT_SPRITE_HUMAN_MALE
+	var/obj/item/organ/external/tail/tail = human_owner.get_organ_slot(ORGAN_SLOT_EXTERNAL_TAIL)
+	if(tail)
+		butt_sprite = BUTT_SPRITE_CAT
```

Comment:
lizard tails are in this slot, so are monkey tails

Reply:
why are we bespoke checking cat here?

Reply:
moved it to the tail level, currently only cat tails will show the cat sprite, unless the chest overrides the proc to show its own sprite